style.css

```diff
@@ -0,0 +1,119 @@
+/* RESET FROM UI */
+
+img {
+  aspect-ratio: 0;
+}
+
+/* HOMEPAGE */
+.heading-container {
+  padding: 1rem 2rem;
+  grid-template-rows: min-content 1fr;
+  grid-template-areas: "header-title nav";
+}
+
+.header-title {
+  align-self: end;
+  grid-area: header-title;
+}
+
+.header-subtitle {
+  margin-left: 0.5rem;
+}
+
+nav {
+  align-self: end;
+  grid-area: nav;
+  margin-left: auto;
+}
+
+.primary-navigation {
+  display: flex;
+  gap: 2em;
+}
+
+.primary-navigation a {
+  color: hsl(var(--clr-dark));
+}
+
+.banner-img {
+  object-fit: contain;
+  height: auto;
+}
+
+.section-container {
+  max-width: 80rem;
+  margin: 0 auto;
+}
+
+.candies-category {
+  gap: 0.25rem;
+}
+
+.subheading {
+  margin-top: 1rem;
+}
+
+.candies-category > * {
+  display: grid;
+}
+
+.bg-image {
+  background-position: right top;
+  object-fit: cover;
+  aspect-ratio: 1.5/1;
+  background-repeat: no-repeat;
+  padding: 1rem;
+}
+
+.section-container {
+  margin-bottom: 2rem;
+}
+
+.candy-name {
+  padding: 1rem;
+}
+
+.candy-name:hover {
+  background: hsl(var(--clr-white) / 0.5);
+  cursor: pointer;
+}
+
+.bg-image-Marshmello {
+  background-image: url(./images/marshmello.jpg);
+}
+
+.bg-image-Gummies {
+  background-image: url(./images/gummies.jpg);
+}
+
+.bg-image-Jellies {
+  background-image: url(./images/jellies.jpg);
+}
+
+.bg-image-Fizzy {
+  background-image: url(./images/fizzy.jpg);
+}
+
+.bg-image-Chocolates {
+  background-image: url(./images/chocolates.jpg);
+}
+
+.bg-image-Lollipops {
+  background-image: url(./images/lollipops.jpg);
```

Comment:
use url string inside quotes for best practice.

Reply:
+1

Reply:
Oh nice! Didnt know this